src/main/java/teammates/ui/template/InstructorFeedbackResultsQuestionTable.java

```diff
@@ -51,22 +49,15 @@ public InstructorFeedbackResultsQuestionTable(InstructorFeedbackResultsPageData
         this.questionStatisticsTable = questionStatisticsHtml;
         this.responses = responseRows;
         
-        this.isQuestionHasResponses = !responses.isEmpty(); //TODO: just check is response is empty in jsp? 
+        this.isQuestionHasResponses = isQuestionHasResponses;  
         
         this.question = question;
         
-        this.questionText = data.getBundle().getQuestionText(question.getId());
+        this.questionText = questionText;
         
-        this.panelClass = responses.isEmpty() ? 
-                          "panel-default" : 
-                          "panel-info";
+        this.panelClass = "panel-info";
```

Comment:
It used to be panel-default if the question has no responses. But now that the responses are always loaded by ajax, the questions always has no responses at first when loading the structure. This panel colour thing based on whether the question has responses will need to be reimplemented in javascript. I'll make an issue for that after this is merged in.